src/step.js

```diff
@@ -22,6 +23,34 @@ export function Step() {
   renderRocket()
   airfighter.render()
 
+  if(mobileVelocity.createdJoystick) {
+    mobileTrafficControl()
+  }
+
   const lastEnemy = enemies[enemies.length - 1]
   levelOverIfLastEnemyOut(lastEnemy, gameState)
 }
+
+function mobileTrafficControl() {
```

Comment:
I think we could have this function in another file.
We can create a folder `src/mobile/` and create a file for this function.
Also we could rename it to something like `setAirfighterVelocityFromMobileJoystick`.

Reply:
[x]